charts/prometheus-pushgateway/values.yaml

```
@@ -358,3 +358,6 @@ networkPolicy: {}
   #   - podSelector:
   #       matchLabels:
   #         app: myapp
+
+# Array of extra K8s objects to deploy (evaluated as a template)
+extraDeploy: []
```

Since a few charts in the repository already support additional manifests, I'd suggest to keep the field's name:

```suggestion
extraManifests: []
```